lib/inline-imports.js

```diff
@@ -86,7 +86,48 @@ function inlineImports(fileContents, options = {}, visited = new Set()) {
   return result.join('\n');
 }
 
+function getFilename(filePath, options = {}) {
+  const { resolveImport, resolveOptions = {} } = options;
+
+  const { basedir } = resolveOptions;
+  if (typeof basedir !== 'string') {
+    throw new Error('inlineImports requires options.resolverOptions.basedir be set');
+  }
+  const resolve = typeof resolveImport === 'function' ? resolveImport : require('resolve').sync;
```

Comment:
This options extraction, validation and defaulting is identical to the code in `linesWithInlinedImportsOf`

Can we extract this to a common utility? `extractOptions` or something